usr/share/lib/ipa/tests/SLES/infra/test_sles_smt_failover.py

```diff
@@ -0,0 +1,27 @@
+import shlex
+
+
+def test_sles_smt_failover(GetSMTServerName,
+                           GetSMTServers,
+                           GetReleaseValue,
+                           host,
+                           request):
+    provider = request.config.getoption('provider')
+    region = request.config.getoption('region')
+
+    result = host.run('cat /etc/hosts | grep %s' % GetSMTServerName(provider))
+    smt_ip = shlex.split(result.stdout)[0]
+
+    pretty_name = GetReleaseValue('PRETTY_NAME')
+    servers = GetSMTServers(pretty_name, provider, region)
+
+    for server in servers:
+        if server['ip'] != smt_ip:
+            result = host.run(
+                "sudo sed -i 's/{current}/{new}/' /etc/hosts".format(
+                    current=smt_ip,
+                    new=server['ip']
+                )
+            )
+            assert result.rc == 0
```

Comment:
This doesn't actually test the failover, the code only changes /etc/hosts to point to a sibling server. The test is to run "zypper ref" to make sure the repos stick around.

Reply:
Yes, this was a bad name for test. The test_sles_smt_sibling.yaml description would be the entire test for failover. Which switches SMT, runs update (ref, up), and then checks the repos.

Will rename the description to test_sles_smt_failover.yaml and this test to test_sles_switch_smt.py. I think that will be better naming given the function of each.